### PR TITLE
Introduce "known issues" page

### PR DESCRIPTION
**Pull request by jaschrep-msft**
Adds the page to datamovement base folder. Readmes for base, blob, and file packages each add a reference to this page as their first H2 section of the document.

### CODE REVIEW

File: sdk/storage/Azure.Storage.DataMovement.Blobs/README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Azure Storage client libraries.
 
 [Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
+## Known Issues
```
Comment (jalauzon-msft) [inline sdk/storage/Azure.Storage.DataMovement.Blobs/README.md:18]: Should we make this even more obvious and move it to the very top with like a "Disclaimer: ..." format? That way it shows up at the top of NuGet too.

File: sdk/storage/Azure.Storage.DataMovement/KnownIssues.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Preserving properties and metadata on copy
+
+When copying data between blobs, blob properties and blob metadata are not preserved. There is currently no support for this feature; properties and metadata can only be defined by the caller to be applied uniformly to each destination blob.
+
+This limitation also exists for share files and directories.
+
+### Client-side encryption support
+
+**There is no support for copying client-side encrypted blobs.** Since client-side encryption is built on blob metadata, the necessary information to decrypt the blob will not be transferred in a service-to-service copy. Azure.Storage.Blobs will not be able to decrypt the copied blob. If that metadata is lost, your blob contents will be lost forever.
```
Comment (jalauzon-msft) [inline sdk/storage/Azure.Storage.DataMovement/KnownIssues.md:13]: One more Known Issue Amanada and I discovered the other day that is probably worth mentioning? Seems trivial to just fix but it's not unfortunately :)

"If you set `DataTransferOptions.InitialTransferSize` larger than 256 MiB for a Block Blob copy, the transfer may fail if the blob being copied is larger than 256 MiB."

File: sdk/storage/Azure.Storage.DataMovement.Blobs/README.md (hunk 1 above)
Comment (schoag-msft) [reply sdk/storage/Azure.Storage.DataMovement.Blobs/README.md:18]: In other repos, we've put the release state in the Title of the readme. E.g. Product name here (Retired) or Product name here (Alpha) or an H1 heading that states "Project status: Beta" (e.g. https://github.com/Azure/azure-cosi-driver#project-status-alpha). Could we do something similar here and would that bubble up to NuGet?

Comment (jaschrep-msft) [reply sdk/storage/Azure.Storage.DataMovement.Blobs/README.md]: I've renamed the section to "Project Status: Beta" and bumped it to the top of the document.

**Review by jalauzon-msft — APPROVED** [review]

**Pull request merged**